_lifeofinternatpuppet/post_7.md

```diff
@@ -41,17 +41,43 @@ On Tuesday 13th October, after finishing the session with Sheena and Ciaran, I w
 
 Why not check this [link out](https://puppet.com/blog/litmus-new-module-acceptance-testing-tool/) to find out more information about Litmus.
 
-The second part of the session with Daiana was on testing a pull request (pr) from the community, my first ticket was assigned to me (changing author name of supported modules to puppetlabs). A to-do-list was given by Daiana on what I learnt in part 2 of the meeting. She also suggested to keep a word document on all the necessary commands used on a day-to-day basis when doing work, She felt having that document really reduces the stress to search for commands within the MAC OS terminal when working on a ticket/issue.
-
-I still have to put together a document which will include all the necessary... stay tuned for it!
+The second part of the session with Daiana was on testing a pull request (pr) from the community, my first ticket was assigned to me (changing author name of supported modules to puppetlabs). A to-do-list was given by Daiana on what I learnt in part 2 of the meeting. Check out what I was working on in the to do list given below:-
+
+Part 1:
+1. Choose a PR from PR reviews report from Community
```

Comment:
1.Choose a PR from PR reviews report from Community
2. Clone the repo of the module locally
3. Change the branch in order to test the changes from the PR, using **git checkout <branch_where_are_the_changes>**

Reply:
Just to be more detailed and specific